common/errors.h

```diff
@@ -370,6 +370,9 @@ enum retcode_t {
 
 typedef enum retcode_t retcode_t;
 
+#define STR_OK "OK"
+
+// Cclient Module
```

Comment:
can we have those in a different file? :)

Reply:
`error_strings.h`?

Reply:
that's what i thought :)